python/cudf/cudf/pandas/__init__.py

```diff
@@ -19,6 +22,46 @@ def install():
     loader = ModuleAccelerator.install("pandas", "cudf", "pandas")
     global LOADED
     LOADED = loader is not None
+    import os
+
+    if (rmm_mode := os.getenv("CUDF_PANDAS_RMM_MODE", None)) is not None:
+        import rmm.mr
+        from rmm.mr import available_device_memory
```

Comment:
Should we import rmm at the top, too? I don’t think we have much benefit to lazy import here.

Also we should use the fully qualified name for rmm.mr.available_device_memory so we only have one import statement for rmm.mr.